.github/workflows/ci.yaml

```diff
@@ -16,14 +17,12 @@ jobs:
       with:
         python-version: "3.x"
     - uses: actions/checkout@v2
-    - name: Lint with black through tox
+    - name: Apply linter
```

Comment:
The advantage before was that you could directly tell on github what exactly failed. Now you would need to go into the github action and check the logs to see what went wrong. I think I slightly preferred the previous behaviour, but I'm fine either way.

Reply:
Actually I'm probably wrong: It was only a sigle github action `lint` anyways so I guess it's the same

Reply:
It was kinda all the same (when the "lint" action fails, you could dig a bit deeper to see which one fails by checking the sub-action.). But now you can see which action fails as well, just in the output of tox instead of as a subaction of the CI. 

I think the improvement is that now, the linter-choice is bundled in a single place: the tox-ini. Before, one implicitly imposes that black and isort _must_ be in there with the specific keys. This makes a bit tedious because everytime a new linter choice is made in tox, a new github subaction needs to be created which is a bit of a duplicate.

This is quite minor and subjective, though. I prefer it bundled, but not very strictly. What do you think? We can also leave it like this and iterate if it bothers us?

Reply:
It seems fine in this way. Thanks :)